tests/deserialzie.rs

```diff
@@ -0,0 +1,14 @@
+#[cfg(test)]
+mod tests {
+	use atspi::events::names::ObjectEvents;
+	use serde_plain;
+
+	#[test]
+	fn serialize_object_variant() {
+		assert_eq!(
```

Comment:
We use it to set up the match rule, see `src/connection.rs` line 177. Of course, we could also set up an infallible `ToString` trait as well. That would probably be better since that reduces our dependencies.